packages/react-native/src/theme/hooks/usePressableStyles.ts

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from 'react';
+import {
+  PressableProps,
+  PressableStateCallbackType,
+  StyleProp,
+  TextStyle,
+  ViewStyle,
+} from 'react-native';
+
+type PressableStyle = ViewStyle | TextStyle;
+
+export type PressableStyleProps = {
+  disabled: PressableProps['disabled'];
+  disabledStyle?: PressableStyle;
+  style: PressableProps['style'];
+  themedStyle: PressableStyle;
+};
```

Comment:
Think we can simplify this to just:
```ts
interface UsePressableStyleParams extends PressableProps<'style'> {}
```

Whether or not something is `disabled` should have no impact on `pressed` styles, which incidentally is also where we are using `themedStyle`